tests/unit/common/test_generators.py

```diff
@@ -0,0 +1,44 @@
+"""Tests classes and functions in boat_simulator/common/generators.py"""
+import math
+
+import numpy as np
+import pytest
+
+from boat_simulator.common.generators import MVGaussianGenerator
+
+
+class TestGaussianGenerator:
+    pass
+
+
+class TestMVGaussianGenerator:
+    @pytest.mark.parametrize(
+        "mean, cov",
+        [
+            (np.array([1, 1]), np.eye(2)),
+            (np.array([1, 1, 1]), np.eye(3)),
+            (np.array([1, 1, 1, 1]), np.eye(4)),
+        ],
```

Comment:
Let's try adding some more tests where:
- The mean isn't just a vector of ones
- The covariance matrix isn't just the identity matrix. When constructing your own covariance matrices, remember the following:
  - If the corresponding mean vector has a shape of `(N,)`, the covariance matrix must be `(N,N)`
  - The covariance matrix must be [positive semi-definite](https://en.wikipedia.org/wiki/Definite_matrix). The underlying functions that you use should assert this, so you can always just try and create a matrix through trial and error until you get one that works rather than constructing one yourself.